Python/Regex/P1.py

```diff
@@ -0,0 +1,14 @@
+"write a pattern for recognizing a legal email address (imagining that the address can only end with .it or .com or .org)"
+import re
+
+
+def email(address:str)->bool:
+   it=r"@(.+?).it$"
```

Comment:
@angelmpalomares maybe this one match also email like that: 
`pippo pluto ==+ @gmail.com`
I don't know if it is a valid one 🤣

Reply:
Falaschi's bad writing hahaha

Reply:
Maybe this one is stronger?
`(\w|\_)+@\w+\.(com)|(it)|(org)`